spec/services/catalog/copy_portfolio_item_spec.rb

```diff
@@ -68,19 +68,43 @@
     end
 
     context "when the portfolio_item has service_plans" do
-      let(:params) { { :portfolio_item_id => portfolio_item.id, :portfolio_id => portfolio.id } }
+      context "when the service_plan base match topology" do
+        let(:params) { { :portfolio_item_id => portfolio_item.id, :portfolio_id => portfolio.id } }
+
+        before do
+          allow(Catalog::SurveyCompare).to receive(:changed?).and_return(false)
+          allow(Catalog::DataDrivenFormValidator).to receive(:valid?).and_return(true)
+
+          portfolio_item.service_plans.create!(
+            :base     => {"schema" => {}},
+            :modified => {"schema" => {}}
+          )
+        end
 
-      before do
-        portfolio_item.service_plans.create(
-          :base => JSON.parse(File.read(Rails.root.join("spec", "support", "ddf", "valid_service_plan_ddf.json")))
-        )
+        it "copies over the service_plan" do
+          new = copy_portfolio_item.new_portfolio_item
```

Comment:
Above you used `copied_portfolio` but here you're using `new`, should we change it to match (and as @syncrou mentioned it's kinda weird, I think maybe you missed this one on accident?)

Reply:
Yep that's my bad. Fixed!